lib/kitchen/verifier/inspec.rb

```diff
@@ -335,6 +336,9 @@ def runner_options_for_winrm(config_data)
           "connection_retry_sleep" => kitchen[:connection_retry_sleep],
           "max_wait_until_ready" => kitchen[:max_wait_until_ready],
         }
+
+        opts["forward_agent"] = config_data[:forward_agent] || kitchen[:forward_agent] if config_data[:forward_agent] || kitchen[:forward_agent]
```

Comment:
I'm not clear on why we'd be forwarding the SSH key over the WinRM connection. Could you explain?

Reply:
Yes I am removing this. It was some confusion on my end.